cmd/saml2aws/commands/configure.go

```diff
@@ -44,19 +44,23 @@ func Configure(configFlags *flags.CommonFlags) error {
 			return errors.Wrap(err, "failed to input configuration")
 		}
 
-		if credentials.SupportsStorage() && idpAccountPassword == "" {
-			password := prompter.Password("Password")
-			if password != "" {
-				if confirmPassword := prompter.Password("Confirm"); confirmPassword == password {
-					idpAccountPassword = password
-				} else {
-					log.Println("Passwords did not match")
-					os.Exit(1)
-				}
-			} else {
-				log.Println("No password supplied")
-			}
+		if !credentials.SupportsStorage() || idpAccountPassword != "" {
+			return nil
 		}
+
+		password := prompter.Password("Enter password")
+		if password == "" {
+			log.Println("No password supplied")
+			return nil
+		}
+
+		repeat := prompter.Password("Re-enter password")
+		if repeat != password {
+			log.Println("Confirmed password does not match")
+			os.Exit(1)
+		}
```

Comment:
is this a good idea? 💭
```suggestion
		for {
			repeat := prompter.Password("Re-enter password")
			if repeat == password {
				break
			}
			log.Println("Confirmed password does not match")
		}
```

Reply:
Agree with repeat. For sign up: maybe? for logins: NO. let's delete?

Reply:
its one time during configure operation.. so I guess if a user gets a password wrong.. we are just assuming they will just do it again.

Seems okay to me.

Reply:
I've squashed - and rebased (sorry @pwmcintyre) - and removed the password repeater.

I couldn't help myself interms of doing some refactoring... I tried to keep as minimal as possible.